data/commands/guide.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
@RussellLVP , Reverted it as per as @BeanerLVP request (ie removed p health and mandatory) . Please comment on this .

Reply:
Specifer, thank you for trying to contribute, but please leave this particular feature for us to deal with, as we're still in the process of discussing it internally.